src/actions/auth.js

```diff
@@ -0,0 +1,69 @@
+import firebase from 'firebase';
+
+const firebaseApp = {
+  apiKey: 'AIzaSyDwnsJcOAy2cP2lyTFdWcanOMTf2DRZpIs',
+  authDomain: 'local-cb336.firebaseapp.com',
+  databaseURL: 'https://local-cb336.firebaseio.com',
+  storageBucket: 'local-cb336.appspot.com',
+  messagingSenderId: '514582937003'
+};
+
+firebase.initializeApp(firebaseApp);
+
+const provider = new firebase.auth.GoogleAuthProvider();
+
+export const startListeningToAuth = () => {
+  return (dispatch, getState) => {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) {
```

Comment:
I fixed the middleware thunk problem. I'm getting that pop-up page where you can deny or accept, but still not changing the state that says the user is logged in.